*** gdal/Driver.cs	2014-12-04 12:14:10.000000000 +0100
--- gdal/Driver.cs.patched	2014-12-04 13:33:38.000000000 +0100
*************** using System.Runtime.InteropServices;
*** 14,20 ****
  public class Driver : MajorObject {
    private HandleRef swigCPtr;
  
!   public Driver(IntPtr cPtr, bool cMemoryOwn, object parent) : base(GdalPINVOKE.DriverUpcast(cPtr), cMemoryOwn, parent) {
      swigCPtr = new HandleRef(this, cPtr);
    }
  
--- 14,20 ----
  public class Driver : MajorObject {
    private HandleRef swigCPtr;
  
!   public Driver(IntPtr cPtr, bool cMemoryOwn, object parent) : base(GdalPINVOKE.Driver_SWIGUpcast(cPtr), cMemoryOwn, parent) {
      swigCPtr = new HandleRef(this, cPtr);
    }
  
